VK/VK/Extensions/TableView+Extension.swift

```diff
@@ -0,0 +1,21 @@
+// TableView+Extension.swift
```

Comment:
```suggestion
// UITableView+Extension.swift
```
Название файла не правильное